File: C/Iniciantes/Uri1019_ConversaoDeTempo.c
#include <stdio.h>

int main(){
	
	int duracao_seg, minutos, horas; 
	
	scanf("%d",&duracao_seg);

	horas = duracao_seg / 3600;
	minutos = (duracao_seg - (horas * 3600)) / 60;
	duracao_seg = (duracao_seg - (horas * 3600) - (minutos * 60));
	
	printf("%d:%d:%d\n", horas, minutos, duracao_seg);
	
	return 0;
}

File: Javascript/Uri1019_ConversaoDeTempo.js
var input = require("fs").readFileSync("stdin", "utf8");

let duracao_seg = parseInt(input);

let horas = parseInt(duracao_seg / 3600);
let minutos = parseInt((duracao_seg - (horas * 3600)) / 60);
duracao_seg = (duracao_seg - (horas * 3600) - (minutos * 60));

console.log(horas + ":" + minutos + ":" + duracao_seg);